pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -475,6 +530,86 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *api.Ser
 		status.Ingress = append(status.Ingress, api.LoadBalancerIngress{IP: floatIP.FloatingIP})
 	}
 
+	if lbaas.opts.ManageSecurityGroups {
+
+		lbSecGroupCreateOpts := groups.CreateOpts{
+			Name:        getSecurityGroupName(clusterName, apiService),
+			Description: fmt.Sprintf("Securty Group for %v Service LoadBalancer", apiService.Name),
+		}
+
+		lbSecGroup, err := groups.Create(lbaas.network, lbSecGroupCreateOpts).Extract()
```

Comment:
Somewhere we need to add a rule to this secgroup to allow (at least) ICMP frag-needed (and perhaps equivalent for IPv6), otherwise PMTUD will be broken.